fenix/app/src/androidTest/java/org/mozilla/fenix/helpers/FeatureSettingsHelperDelegate.kt

```diff
@@ -30,7 +30,7 @@ class FeatureSettingsHelperDelegate : FeatureSettingsHelper {
         isRecentTabsFeatureEnabled = settings.showRecentTabsFeature,
         isRecentlyVisitedFeatureEnabled = settings.historyMetadataUIFeature,
         isPWAsPromptEnabled = !settings.userKnowsAboutPwas,
-        isTCPCFREnabled = settings.shouldShowTotalCookieProtectionCFR,
+        isTCPCFREnabled = settings.shouldShowTotalCookieProtectionCFR && settings.shouldShowCookieBannerReEngagementDialog(),
```

Comment:
I think this would normally be
```suggestion
        isTCPCFREnabled = settings.shouldShowTotalCookieProtectionCFR && !settings.shouldShowCookieBannerReEngagementDialog(),
```
the same as in the `BrowserToolbarCFRPresenter.kt`
but right now there is no way to show / considered as already shown the CBH and that's why these were combined this way?

I'm just thinking that if so then these lines don't need any change?
cc @AndiAJ also.